tic-tac-toe/src/components/game/game.js

```diff
@@ -0,0 +1,124 @@
+import React, {Component} from 'react';
+import Board from '../board/board';
+import ControlPannel from '../control-pannel/control-pannel';
+import GameInfo from '../info-bar/info-bar';
+import InfoBlock from '../info-block/info-block';
+import checkTurn from '../../scripts/calculate-next-move';
+import calculateWinner from '../../scripts/calculate-winner';
+import {getGameData, updateGameData} from '../../scripts/storage-controller';
+import './game.css';
+
+class Game extends Component {
+    constructor(props) {
+		super(props);
+    this.isBlockGame = false;
+		this.state = getGameData() || {
+			history: [
+				{
+					squares: Array(9).fill(null),
+				}
+			],
+			stepNumber: 0,
+			xIsNext: true,
+      activeGameMode: 'human',
```

Comment:
create dictionary to keep game modes

Reply:
I fixed it with creation library.js
```suggestion
      stepNumber: 0,
      xIsNext: true,
      activeGameMode: gameModes.vsPlayer,
```